### ISSUE TITLE
[Tweak] Finger Photo Vertical Mode

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [Frooxius/NeosPublic#1372]
As title. I think a vertical mode would be neat to have on the FingerPhoto gesture.
Something like rotating both hands by 90° towards one or the other side.

**Comment by shiftyscales** [comment]
Implementing something like this would likely require reworking the gesture entirely, similar to #1179.

There also isn't currently a concept of portrait vs landscape photos, but that in of iself could be done just by swapping the x and y values of the camera.

**Comment by Frooxius** [comment]
Yeah this would take some time to implement, because it adds more ambiguity to interpreting the gesture and thus more potential for misbehavior. It can be added at some point, but I'd consider this low priority right now.

**Comment by 3x1t-5tyl3** [comment]
Updated the text to be more inline with modern issue standard.
